bluebottle/payments_flutterwave/adapters.py

```diff
@@ -18,15 +20,44 @@
 
 
 class FlutterwavePaymentAdapter(BasePaymentAdapter):
-    MODEL_CLASSES = [FlutterwavePayment]
+    MODEL_CLASSES = [FlutterwavePayment, FlutterwaveMpesaPayment]
```

Comment:
We now we use the psp name to determine what adapter to use. We do the same with Docdata CC/Ideal vs Docdata Directdebit. I totally agree that this would be more readable if we would have separate adapters. Should we then split it and have adapters called DocdataDirectdebit, DocdataCreditcard and so on?